### PR TITLE
Add a TexLive package

### PR DESCRIPTION
**Pull request by rnauber**
This patch packages the tex binaries (pdftex, luatex, ...) and an installer ("termux-install-tl"). The latter fetches an initial set of the texlive distribution. Further packages can be fetched through "tlmgr".  I am happy to get some feedback! This relates to issue #171. 

See you,
Richard

### CODE REVIEW

**Comment by rnauber** [comment]
I fixed that! 
Here is a video of texlive in action:
https://asciinema.org/a/bkch63bpio5xkqtlryfglnsbc

File: packages/icu/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=http://www.gnu.org/software/libidn/
+TERMUX_PKG_DESCRIPTION="GNU Libidn library, implementation of IETF IDN specifications"
```
Comment (fornwall) [inline packages/icu/build.sh]: TERMUX_PKG_HOMEPAGE  and TERMUX_PKG_DESCRIPTION needs to be updated for icu.

File: packages/icu/fix_makefile_install.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- ./Makefile.in~	2016-09-09 23:28:18.000000000 +0200
++++ ./Makefile.in	2016-12-20 18:22:04.574323868 +0100
+@@ -180,7 +180,7 @@
+ 	$(INSTALL_SCRIPT) $(top_srcdir)/install-sh $(DESTDIR)$(pkgdatadir)/install-sh
+ 	@$(MKINSTALLDIRS) $(DESTDIR)$(libdir)/pkgconfig
+ 	$(INSTALL_DATA) $(ALL_PKGCONFIG_FILES) $(DESTDIR)$(libdir)/pkgconfig/
+-	$(INSTALL_DATA) $(top_srcdir)/../LICENSE $(DESTDIR)$(pkgdatadir)/LICENSE
++#	$(INSTALL_DATA) $(top_srcdir)/../LICENSE $(DESTDIR)$(pkgdatadir)/LICENSE
```
Comment (fornwall) [inline packages/icu/fix_makefile_install.patch]: Hm, why is this necessary?

File: packages/poppler/build.sh [hunk 3]
```diff
@@ -4,4 +4,4 @@ TERMUX_PKG_VERSION=0.47.0
 TERMUX_PKG_SRCURL=http://poppler.freedesktop.org/poppler-${TERMUX_PKG_VERSION}.tar.xz
 # libcairo and littlecms is used by pdftocairo:
 TERMUX_PKG_DEPENDS="fontconfig, libcairo, libpng, libjpeg-turbo, libtiff, littlecms, openjpeg"
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-poppler-glib"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-poppler-glib  --enable-xpdf-headers"
```
Comment (fornwall) [inline packages/poppler/build.sh]: I guess the old xpdf headers are needed by texlive? Could you add a comment above this field stating that?

File: packages/icu/fix_makefile_install.patch (hunk 2 above)
Comment (rnauber) [inline packages/icu/fix_makefile_install.patch]: the toplevel directory (where LICENSE is located) will not be extracted...

**Comment by rnauber** [comment]
Hi @fornwall , 
The PR now uses the libicu from @vishalbiswas and I think it is now ready to merge, do you agree?
See you,
RIchard

**Comment by fornwall** [comment]
@rnauber Thanks a lot! I'm working on the package to enable system harfbuzz-icu and harfbuzz with graphite support, will get back to you!

**Comment by rnauber** [comment]
Great, thanks for merging it!
